config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create]do
+    resources :comments , only:[:index, :new, :create]
+    resources :likes , only:[:create]
+    end
   end
 
   resources :posts, only: [:index, :show]
```

Comment:
- [ ] Kindly remove this resources (`resources :posts, only: [:index, :show]`) from your routes.rb file. It already defined in the block above.

Reply:
Removed :)